### PR TITLE
Add support for `--fields=+l`

### PR DESCRIPTION
**Pull request by jackkinsella**
See issue #81 for discussion on motivation.

### CODE REVIEW

File: lib/ripper-tags.rb [hunk 1]
```diff
@@ -109,6 +109,9 @@ def self.option_parser(options)
       opts.on("-n", "Equivalent to --excmd=number.") do
         options.excmd = "number"
       end
+      opts.on("--fields=+l", "Include language information in the tag") do |flags|
```
Comment (mislav) [inline lib/ripper-tags.rb]: There's already a block that processes the `--fields` option: `opts.on("--fields=+n"`. There doesn't have to be a new one. You can change its signature to `opts.on("--fields=+nl"` (to demonstrate the two flags that we support) and change its description to something like “add extra fields to output; see `ctags --list-fields` for explanation”.

Comment (jackkinsella) [reply lib/ripper-tags.rb]: Good point! I wrongly assumed that `opts.on`'s first argument was an exact match thing.

File: lib/ripper-tags.rb [hunk 2]
```diff
@@ -109,7 +109,7 @@ def self.option_parser(options)
       opts.on("-n", "Equivalent to --excmd=number.") do
         options.excmd = "number"
       end
-      opts.on("--fields=+n", "Include line number information in the tag") do |flags|
+      opts.on("--fields=+ln", "Add extra fields to output") do |flags|
```
Comment (jackkinsella) [inline lib/ripper-tags.rb]: Using alphabetical ordering.

**Comment by mislav** [comment]
Thank you!

**Pull request merged**